src/components/status/status.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { expect, fixture } from '@open-wc/testing';
```

Comment:
- Use `.toEqualSnapshot()` to compare html in DOM/ShadowDOM
- Maybe add tests for all the variants?

Reply:
with [c8edefd](https://github.com/lyne-design-system/lyne-components/pull/2262/commits/c8edefd600da53ac3414f3271df11f686b1e6718)
- added snapshot
- added tests for each type

Reply:
Just a note: `data-has-title` is automatically set in the render method, so it can be removed in all the fixtures, using `extended` should be enough